app/controllers/games_controller.rb

```diff
@@ -0,0 +1,34 @@
+class GamesController < ApplicationController
+
+  def new
+    @game = Game.new
+  end
+
+  def index
+  end
+
+  def create
+    @game = Game.create(game_params)
+    @game.update_attribute(white_player_id, current_user.id)
+    redirect_to root_path
+  end
+
+  def show 
+    @game = Game.find_by_id(params[:id])
+    if @game.blank?
+      render text: 'Not Found', status: :not_found
+    end
```

Comment:
Fair enough. There's plenty of room here for personal/team preference. I find the single line, trailing conditionals pretty elegant compared to the standard version and tend to use them a lot, but I can totally see the concerns around clarity, especially when methods without parens are also involved.